docs/_docs/persistence/snapshots.adoc

```diff
@@ -160,27 +166,27 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 == Consistency Guarantees
 
 All snapshots are fully consistent in terms of concurrent cluster-wide operations as well as ongoing changes with Ignite
-Persistence and other files on nodes.
+Persistence and files on nodes.
 
 The cluster-wide snapshot consistency is achieved by triggering the link:https://cwiki.apache.org/confluence/display/IGNITE/%28Partition+Map%29+Exchange+-+under+the+hood[Partition-Map-Exchange]
 procedure. By doing that, the cluster will eventually get to the point in time when all previously started transactions are completed, and new
-ones are paused. Once this happens, the cluster initiates the snapshot creation procedure.
+ones are paused. Once this happens, the cluster initiates the snapshot creation procedure. Primary and backup cache partitions
```

Comment:
The "cache" is redundant in the sentence. We can simply say primary and backup partitions. Plus, some people work with SQL tables and this reference to caches can cause confusion.

Consider this version, it fixes some misprints and addresses the point above:

Once this happens, the cluster initiates the snapshot creation procedure. The PME procedure ensures that the snapshot includes primary and backup in a consistent state.

Reply:
Fixed.